package.json

```diff
@@ -49,7 +49,8 @@
     "nodemon": "^2.0.20",
     "redis-mock": "^0.56.3",
     "stylelint": "^14.9.1",
-    "stylelint-config-standard": "^26.0.0"
+    "stylelint-config-standard": "^26.0.0",
+    "testdouble": "^3.16.8"
```

Comment:
Do we want to include this as part of `src/package.json` or is intended to live in the root (legacy) package?  (We _will_ have to reconcile both files, but it might be nice for new stuff to live in the new package?)